elements/pfe-accordion/demo/pfe-accordion.html

```diff
@@ -1,221 +1,188 @@
+        et velit neque. Donec ultricies condimentum mauris, pellentesque imperdiet libero convallis convallis. Aliquam
+        erat volutpat. Donec rutrum semper tempus. Proin dictum imperdiet nibh, quis dapibus nulla. Integer sed tincidunt
+        lectus, sit amet auctor eros.
+      </p>
+      <p>
         Donec vel posuere orci. Phasellus quis tortor a ex hendrerit efficitur. Aliquam lacinia ligula pharetra,
-        sagittis ex
-        ut, pellentesque diam. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae;
+        sagittis ex ut, pellentesque diam. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae;
         Vestibulum ultricies nulla nibh. Etiam vel dui fermentum ligula ullamcorper eleifend non quis tortor. Morbi
-        tempus
-        ornare tempus. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Mauris
-        et
-        velit neque. Donec ultricies condimentum mauris, pellentesque imperdiet libero convallis convallis. Aliquam
-        erat
-        volutpat. Donec rutrum semper tempus. Proin dictum imperdiet nibh, quis dapibus nulla. Integer sed tincidunt
-        lectus,
-        sit amet auctor eros.
-      </div>
-
+        tempus ornare tempus. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Mauris
+        et velit neque. Donec ultricies condimentum mauris, pellentesque imperdiet libero convallis convallis. Aliquam
+        erat volutpat. Donec rutrum semper tempus. Proin dictum imperdiet nibh, quis dapibus nulla. Integer sed tincidunt
+        lectus, sit amet auctor eros.
+      </p>
     </pfe-accordion-panel>
     <pfe-accordion-header>
       <h3>Item five</h3>
     </pfe-accordion-header>
     <pfe-accordion-panel>
-      Vivamus finibus dictum ex id ultrices. Mauris dictum neque a iaculis blandit.
+      <p>Vivamus finibus dictum ex id ultrices. Mauris dictum neque a iaculis blandit.</p>
     </pfe-accordion-panel>
   </pfe-accordion>
 </article>
 
 
 <article>
-  <h2>Bordered Accordion Example</h2>
-  <input type="checkbox" id="toggle-size" name="toggle-size">
-  <label for="toggle-size">Display Large Size</label>
+  <header>
+    <h2>Bordered</h2>
+    <pfe-switch id="toggle-size" name="toggle-size"></pfe-switch>
+    <label for="toggle-size" data-state="on">Size: Large</label>
+    <label for="toggle-size" data-state="off">Size: Normal</label>
+  </header>
+
   <pfe-accordion id="bordered-example" bordered>
     <pfe-accordion-header>
       <h3>Item one</h3>
     </pfe-accordion-header>
     <pfe-accordion-panel>
-      Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore
-      magna
-      aliqua.
+      <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
```

Comment:
Noticing something here with the `large` attribute turned on, the `::after` border is not going up the whole way on the element.  I can see two ways currently that fixes it, one is with removing the `<p></p>` tags around the content inside the panel and the other is removing the `--pf-c-accordion__panel-body--PaddingTop: 0` inside `pfe-accordion-panel.scss`.

Reply:
Content should be structured for semantics

We could remove margin block start on first child, block end on last child, but that feels like overreach to me

WDYT @marionnegp ?

Reply:
I'd like to merge this and rh-accordion asap, so if this isn't a blocker, please drop an approval here